deploy/kubernetes/metricbeat-kubernetes.yaml

```diff
@@ -353,6 +353,11 @@ rules:
   - nodes/stats
   verbs:
   - get
+# Comment out if apiserver metricset is enabled
```

Comment:
Should we add the `apiserver` metricset to the list of metricsets in `kubernetes.yml`? (Commented out if we don't think this should be enabled by default)